src/vs/workbench/browser/parts/views/treeView.ts

```diff
@@ -1442,9 +1440,9 @@ export class CustomTreeViewDragAndDrop implements ITreeDragAndDrop<ITreeItem> {
 
 	onDragOver(data: IDragAndDropData, targetElement: ITreeItem, targetIndex: number, originalEvent: DragEvent): boolean | ITreeDragOverReaction {
 		const types: Set<string> = new Set();
-		originalEvent.dataTransfer?.types.forEach((value, index) => {
-			if (INTERNAL_MIME_TYPES.indexOf(value) < 0) {
-				types.add(this.convertKnownMimes(value).type);
+		originalEvent.dataTransfer?.types.forEach((type) => {
+			if (INTERNAL_DND_MIME_TYPES.indexOf(type) < 0) {
+				types.add(type === DataTransfers.RESOURCES.toLowerCase() ? Mimes.uriList : type);
 			}
 		});
 
```

Comment:
Something in this change is adding extra mime types that shouldn't be added. The following line logs the mime types. I tested with the tree view sample by dragging one of the tree items onto another. In insiders, the following mime types are listed:
```
TreeView dragged mime types: text/plain, text/uri-list, application/vnd.code.tree.testviewdraganddrop
```
With this change:
```
TreeView dragged mime types: text/plain, text/uri-list, codefiles, application/vnd.code.tree.testviewdraganddrop, codeeditors
```

`codefiles` and `codeeditors` should not appear in this list.

Reply:
Good catch as I believe this is also an issue with the drop into editor and copy paste api. Will see if I can fix this in shared code